backend/datacollector/models.py

```diff
@@ -13,4 +13,4 @@ class Message(models.Model):
     color = models.CharField(max_length=20)
     original_message = models.TextField()
     generated_verse = models.TextField()
-    timestamp = models.DateTimeField()
+    timestamp = models.DateTimeField(auto_now_add=True)
```

Comment:
can we have a more descriptive name on this field? timestamp is a data type. is it something like `created_at` or `updated_at`?